pkg/sdk/sdk.go

```diff
@@ -101,5 +102,8 @@ func NewPOKTRollSDK(ctx context.Context, config *POKTRollSDKConfig) (POKTRollSDK
 		return nil, fmt.Errorf("failed to decode private key: %w", err)
 	}
 
+	// Start the ring cache.
+	sdk.ringCache.Start(ctx)
```

Comment:
When does the SDK stop it?

Reply:
It doesn't but the goroutine it starts receives a cancellable context that calls stop

Reply:
#PUC